.xo-config.js

```diff
@@ -50,7 +43,14 @@ module.exports = {
 				svg: 'always'
 			}
 		],
-		'unicorn/no-fn-reference-in-iterator': 'off'
+
+		// Temporarily disabled
+		'@typescript-eslint/no-invalid-void-type': 'off',
+		'@typescript-eslint/no-extra-non-null-assertion': 'off',
```

Comment:
This one wanted to fix something that TypeScript didn't agree with. I suggest keeping it disabled for a while and hope it will be improved.

Reply:
The rule wants us to convert:

```js
if (checksCounter?.textContent!.trim() === '0') {
	checksCounter?.parentElement!.remove();
}
```

to

```js
if (checksCounter!.textContent!.trim() === '0') {
	checksCounter!.parentElement!.remove();
}
```

Are we ok with that?

Reply:
Not ok, `checksCounter` might actually be missing, so this would cause an error.

The condition should read as "if it exists and it's equal to 0, remove it"

Reply:
The only unnecessary part in that is the second `checksCounter?` since if the condition is true, `checksCounter` is definitely not `undefined`. But TypeScript doesn't recognize that yet.

Reply:
Yeah, I'm disabling this rule for now.